Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/magic.yml

```diff
@@ -111,3 +111,50 @@
       types:
         Piercing: 300
     ignoreResistances: true
+
+- type: entity
+  id: ProjectileFingerGun
+  name: mime bullet
+  description: insert raunchy finger bang joke
```

Comment:
Nobody'll see this but I still feel it should be properly capitalised, like:
`Insert raunchy finger bang joke here.`